### PR TITLE
update oscap-vm manual page

### PR DESCRIPTION
**Pull request by jan-cerny**
The manual page for oscap-vm is very short. This pull request tries to improve the manual page.

### CODE REVIEW

**Comment by jan-cerny** [comment]
Any feedback is welcome.

File: utils/oscap-vm.8 [hunk 1]
```diff
@@ -1,70 +1,157 @@
...
 Please report bugs using https://github.com/OpenSCAP/openscap/issues
 
+.SH SEE ALSO
+oscap(8), scap-security-guide(8)
+
+For detailed information please visit OpenSCAP website: https://www.open-scap.org
+
 .SH AUTHORS
 .nf
 Martin Preisler <mpreisle@redhat.com>
```
Comment (mpreisler) [inline utils/oscap-vm.8]: add yourself here

**Comment by mpreisler** [comment]
I will add Jan's name after this is merged to not hold up the merge.

**Comment by mpreisler** [comment]
ACK

**Comment by mpreisler** [comment]
Added Jan in ac71aecad2792c94cf4842cc2ada83e9c908756f

**Comment by jan-cerny** [comment]
@mpreisler Thank you

**Pull request merged**